examples/functions/declare_function.enaml

```diff
@@ -0,0 +1,43 @@
+#------------------------------------------------------------------------------
+# Copyright (c) 2013, Nucleic Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#------------------------------------------------------------------------------
+""" An example showing how to declare a declarive function using func
+
+This example demonstrates how one can use the `func` keyword to declare
+a function bound to an enaml declaration and how scoping rules apply.
+
+Note that when declaring a function using the `func` it will always be
+considered a method of the object on which it is defined hence the possibility
+to access to self which is a reference to the main window.
```

Comment:
can you change this to "to access `self` which in this case is a reference to the main window".